chromatic/rainbows/readers/eureka_specdata.py

```diff
@@ -59,11 +59,11 @@ def from_eureka_SpecData(rainbow, filepath, optimal=True):
 
     # populate a 1D array of times (with astropy units of time)
     k = "time"
-    if dataset[k].attrs["time_units"] == "BJD_TDB":
+    if dataset[k].attrs["time_units"] == "BMJD_TDB":
         astropy_times = Time(dataset[k].data, format="mjd", scale="tdb")
         rainbow.set_times_from_astropy(astropy_times, is_barycentric=True)
     else:
-        t_unit = u.Unit(dataset[t_key].attrs["time_units"])
+        t_unit = u.Unit(dataset[k].attrs["time_units"])
```

Comment:
Good catch. I think this block never got run with the older version, but with the mismatched names it appeared.